### ISSUE TITLE
Mobs resetting in combat

### ISSUE THREAD

**Issue by VoidLock92** [dalaranwow/dalaran-wow#5630]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Mobs are resetting when me and my friend are fighting the same one with spells from a distance. Not all mobs behave this way it seems to be when we are uphill or downhill from them and we both keep taking threat from each other while casting on him. 

**Expected behaviour**: The mob should have kept fighting us and not evaded and reset during combat.

**Steps to reproduce the problem**:

1. Have 2 players attack the same mob with spells from a range.
2. Keep fighting more mobs while casting eventually some of them reset and go back to full health.
3. Get frustrated

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
